monitoring/lifecycle/dashboard.py

```diff
@@ -1,257 +1,764 @@
+                legendFormat='{{' + label + '}}',
             ),
+        ] + ([
             Target(
-                expr=f'sum(rate(s3_lifecycle_s3_operations_total{{{errorLabel}}}[$__rate_interval]))',
-                legendFormat="error",
+                expr='histogram_quantile(0.95, sum(rate(' + GcMetrics.DURATION.bucket() + ')) by (le))',
+                legendFormat="gc",
             ),
-        ],
+        ] if title == 'Duration by Type' else []),
     )
+    for title, metric, label in [
+        ('Trigger Latency',      Metrics.TRIGGER_LATENCY, 'type'),
+        ('Latency over time',    Metrics.LATENCY,         'type'),
+        ('Duration by Type',     Metrics.DURATION,        'type'),
+        ('Duration by Location', Metrics.DURATION,        'location'),
+    ]
+]
 
+latency_distribution, expiration_distribution, transition_distribution, archive_distribution, restore_distribution = [
+    Heatmap(
+        title=title,
+        dataSource='${DS_PROMETHEUS}',
+        dataFormat='tsbuckets',
+        hideZeroBuckets=True,
+        maxDataPoints=25,
+        tooltip=Tooltip(show=True, showHistogram=True),
+        yAxis=YAxis(format=UNITS.SECONDS, decimals=0),
+        cards={'cardPadding': 1, 'cardRound': 2},
+        color=HeatmapColor(mode='opacity'),
+        targets=[Target(
+            expr='sum(increase(' + metric + ')) by(le)',
+            format="heatmap",
+            legendFormat="{{le}}",
+        )],
+    )
+    for title, metric in [
+        ('Latency Distribution',             Metrics.LATENCY.bucket()),
+        ('Expiration Duration Distribution', Metrics.DURATION.bucket(type='expiration')),
+        ('Transition Duration Distribution', Metrics.DURATION.bucket(type='transition')),
+        ('Archive Duration Distribution',    Metrics.DURATION.bucket(type='archive')),
+        ('Restore Duration Distribution',    Metrics.DURATION.bucket(type='restore')),
+    ]
+]
 
-def kafka_messages_time_series(title, expr):
-    return TimeSeries(
+workflow_by_location, workflow_by_type = [
+    PieChart(
         title=title,
+        dataSource='${DS_PROMETHEUS}',
+        legendDisplayMode='hidden',
+        pieType='donut',
+        reduceOptionsCalcs=['mean'],
+        unit=UNITS.OPS_PER_SEC,
+        targets=[
+            Target(
+                expr='sum(rate(' + Metrics.DURATION.count() + ')) by ('+ label + ')',
+                legendFormat='{{' + label + '}}',
+            ),
+        ]
+    )
+    for title, label in [
+        ("By location", 'location'),
+        ("By type",     'type'),
+    ]
+]
+
+s3_delete_object_ops, s3_delete_mpu_ops = [
+    TimeSeries(
+        title=f'{op} Request Rate',
         dataSource="${DS_PROMETHEUS}",
         fillOpacity=5,
-        scaleDistributionType='log',
-        scaleDistributionLog=10,
-        legendDisplayMode='hidden',
+        unit=UNITS.REQUESTS_PER_SEC,
+        stacking={"mode": "normal", "group": "A"},
```

Comment:
For consistency with other graphs
```suggestion
        unit=UNITS.REQUESTS_PER_SEC,
        stacking={"mode": "normal", "group": "A"},
        lineInterpolation='smooth',
```